src/Map/doc/index.rst

```diff
@@ -0,0 +1,285 @@
+Symfony UX Map
+==============
+
+**EXPERIMENTAL** This component is currently experimental and is likely
+to change, or even change drastically.
+
+Symfony UX Map is a Symfony bundle integrating interactive Maps in
+Symfony applications. It is part of `the Symfony UX initiative`_.
+
+Installation
+------------
+
+.. caution::
+
+    Before you start, make sure you have `StimulusBundle configured in your app`_.
+
+Install the bundle using Composer and Symfony Flex:
+
+.. code-block:: terminal
+
+    $ composer require symfony/ux-map
+
+If you're using WebpackEncore, install your assets and restart Encore (not
+needed if you're using AssetMapper):
+
+.. code-block:: terminal
+
+    $ npm install --force
+    $ npm run watch
+
+    # or use yarn
+    $ yarn install --force
+    $ yarn watch
+
+After installing the bundle, ensure the line ``{{ ux_map_script_tags() }}`` is present in your Twig template, e.g.:
+
+.. code-block:: twig
+
+        {% block javascripts %}
+            {% block importmap %}{{ importmap('app') }}{% endblock %}
+            {{ ux_map_script_tags() }}
+        {% endblock %}
+        
+Configuration
+-------------
+
+Configuration is done in your ``config/packages/ux_map.yaml`` file:
+
+.. code-block:: yaml
+    
+    # config/packages/ux_map.yaml
+    ux_map:
+      provider: '%env(UX_MAP_DSN)%'
+
+The ``UX_MAP_DSN`` environment variable should contain the provider DSN to use, e.g. ``google-maps://`` or ``leaflet://``.
+See :ref:`Available providers` for more information on the available providers.
+
+Usage
+-----
+
+Creating and rendering
+~~~~~~~~~~~~~~~~~~~~~~
+
+To create a map, you must use the ``MapFactoryInterface`` service. 
+This service allows you to create a new ``Map`` instance by using the default configured provider, 
+or a different ``ProviderInterface`` implementation if necessary::
```

Comment:
Not to mention that early. Let's focus on the basic standard case, and add later in the document (if necessary) ... i really don't think "coding its own provider" will concern many people ;)